paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetCustomerSessionTest.kt

```diff
@@ -0,0 +1,169 @@
+package com.stripe.android.paymentsheet
+
+import com.stripe.android.core.utils.urlEncode
+import com.stripe.android.networktesting.RequestMatchers.bodyPart
+import com.stripe.android.networktesting.RequestMatchers.host
+import com.stripe.android.networktesting.RequestMatchers.method
+import com.stripe.android.networktesting.RequestMatchers.path
+import com.stripe.android.networktesting.testBodyFromFile
+import com.stripe.android.paymentsheet.utils.IntegrationType
+import com.stripe.android.paymentsheet.utils.PaymentSheetTestRunnerContext
+import com.stripe.android.paymentsheet.utils.TestRules
+import com.stripe.android.paymentsheet.utils.assertCompleted
+import com.stripe.android.paymentsheet.utils.runPaymentSheetTest
+import org.junit.Rule
+import org.junit.Test
+
+class PaymentSheetCustomerSessionTest {
+    @get:Rule
+    val testRules: TestRules = TestRules.create()
+
+    private val composeTestRule = testRules.compose
+    private val networkRule = testRules.networkRule
+
+    private val page: PaymentSheetPage = PaymentSheetPage(composeTestRule)
+
+    @Test
+    fun allowRedisplayIsUnspecifiedWhenNotSavingWithPaymentIntent() = runPaymentSheetTest(
+        networkRule = networkRule,
+        integrationType = IntegrationType.Compose,
```

Comment:
should `integrationType` have a default value here so that we don't have to always specify it? It doesn't seem relevant to these tests